lib/cartopy/io/img_tiles.py

```diff
@@ -577,19 +577,18 @@ def __init__(self,
                          cache=cache)
         self.apikey = apikey
 
-        if layer not in ['Outdoor', 'Road', 'Light', 'Night', 'Leisure']:
+        if layer not in ("Road_3857", "Outdoor_3857", "Light_3857",
+                         "Road", "Outdoor", "Light"):
             raise ValueError(f'Invalid layer {layer}')
+        elif layer in ("Road", "Outdoor", "Light"):
+            layer += "_3857"
```

Comment:
I agree that adding support for EPSG:27700 would be great, and also that it would be a fair bit more work to get it running. I can add a new feature request to the backlog and get that done when I have some time to get more into Cartopy's code.